src/23680.c

```diff
@@ -231,21 +237,24 @@ void spawn_drops(Enemy* enemy) {
         }
     }
 
+    // determine number of coins to drop
+
     itemToDrop = ITEM_COIN;
-    do {} while (0);
-    minCoinBonus = drops->minCoinBonus;
-    tempMax = drops->maxCoinBonus;
 
+    //TODO maybe use an ASSERT here and forgo the odd support for reversing min/max
```

Comment:
Maybe debug_printf a warning instead

Reply:
I dont *think* this is used by any vanilla enemies, would need to check. I think its pretty unintuitive and probably just for sanity checking, which I think an assert would be appropriate for.